src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReferralHelper.java

```diff
@@ -524,14 +524,15 @@ public static List<String> validateEmail(String email) {
     /**
      * Generates a map of personalised information that will be used for the
      * placeholder fields in the Referral email template.
-     * @param detail Contains all the case details.
+     * @param details Contains all the case details.
      * @param referralNumber The number of the referral.
      * @param username Name of the user making or replying to this referral.
+     * @param linkToExui link to Exui with the caseId as a parameter
      * @param isNew Flag for if this is a new referral.
      */
-    public static Map<String, String> buildPersonalisation(CaseDetails detail, String referralNumber, boolean isNew,
-                                                           String username) {
-        CaseData caseData = detail.getCaseData();
+    public static Map<String, String> buildPersonalisation(CaseDetails details, String referralNumber, boolean isNew,
+                                                           String username, String linkToExui) {
```

Comment:
minor: do we need to pass in CaseDetails now? looks like passing CaseData would suffice